content/blog/2023/11/2023-11-17-twim.md

```diff
@@ -31,6 +31,7 @@ image = "https://matrix.org/blog/img/matrix-logo.png"
 > * [MSC4077: Improved process for handling deprecated HTML features](https://github.com/matrix-org/matrix-spec-proposals/pull/4077)
 > * [MSC4076: Let E2EE clients calculate app & E2EE-room badge counts themselves](https://github.com/matrix-org/matrix-spec-proposals/pull/4076)
 > 
+<!-- more -->
```

Comment:
It seems to be in a rather odd place for TWIM, breaking a section in the middle :)

Reply:
Had some doubts about it too but figured the section was too big for a preview and wouldn't hurt to break it but I'm fine with having it move to the end.